src/Cake.Issues.Markdownlint.Tests/LogFileFormat/MarkdownlintLogfileHelperTests.cs

```diff
@@ -0,0 +1,59 @@
+﻿namespace Cake.Issues.Markdownlint.Tests.LogFileFormat
+{
+    using Cake.Issues.Markdownlint.LogFileFormat;
+    using Cake.Issues.Testing;
+    using Shouldly;
+    using Xunit;
+
+    public sealed class MarkdownlintLogFileHelperTests
+    {
+        public sealed class TheCtor
+        {
+            [Fact]
+            public void GetRegexForMarkdownIssueLine_Should_Return_Correct_Pattern_For_Markdownlint_Cli_Smaller_Than_0_19_0()
+            {
+                // Given / When
```

Comment:
Add `// Given` comment for the `var issueLine = ...` and `// When` comment for the `var result =...` line. (Same for other test cases)